docs/concepts/metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+id: metrics
+title: Temporal Metrics
+description: Temporal and Workers both emit metrics| this page describes them
+sidebar_label: Metrics
+---
+
+## Worker/SDK Metrics
+
+The Temporal SDKs emit a largely common set of metrics. Note that all the below metrics are prefixed
+with `temporal_` before being exported to their configured destination(s). `Core`
```

Comment:
```suggestion
Each [Worker Entity](/docs/content/what-is-a-worker-entity) emits metrics.
Each SDK varies in which metrics are emitted.

All the metrics listed below are prefixed
with `temporal_` before being exported to their configured destination(s).
```
Not sure what `Core` is for there?

Reply:
Forgot to finish that sentence. Fixed. Not a huge fan of "worker entity" here as the SDK metrics include things that aren't necessarily tied to a specific worker, though most are.

Reply:
How would I know the difference?
Seems like we could identify, for each metric, where it is emitted from - i.e Worker Entity or Temporal Client?